pkg/virt-api/rest/subresource.go

```diff
@@ -1108,22 +1045,16 @@ func (app *SubresourceAPIApp) addVolumeRequestHandler(request *restful.Request,
 		return
 	}
 
-	opts := &v1.AddVolumeOptions{}
-	if request.Request.Body != nil {
-		defer request.Request.Body.Close()
```

Comment:
according to https://github.com/golang/go/commit/2ede818ae0761fc57dc1b52b4c8673933a3ff3fa there is no need to close the request body.